packages/app/src/cli/services/context/identifiers-extensions.test.ts

```diff
@@ -1150,22 +1150,34 @@ describe('ensureNonUuidManagedExtensionsIds: for extensions managed in the TOML'
 
   test('if there are possible matches, creates the extension for those that dont match', async () => {
```

Comment:
That's implicitly tested here. `if there are possible matches, creates the extension for those that dont match` and what's implicit is "and doesn't create the extension for those that do". and the `config` object here is the one that matches because it's a remote module and also a local one